tests/Tests.fs

```diff
@@ -6,6 +6,7 @@ module Elmish.UrlParserTests
 open Swensen.Unquote
 open Elmish.UrlParser
 
+[<StructuralEquality; NoComparison>]
```

Comment:
Why do we need these?

Reply:
I thought it was required but apparently not. let's see the ci after the removal